<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ECC Cipher</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_gen_keys">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Signature</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_info">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>521</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txt_sign">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>521</width>
     <height>87</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Keys</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sign">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verify">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
